mino/minogrpc/server.go

```diff
@@ -135,20 +145,27 @@ func (rpc RPC) Call(req proto.Message,
 	return out, errs
 }
 
-// Stream implements mino.RPC.Stream()
+// Stream implements mino.RPC.Stream.
```

Comment:
I was about to make a PR to change the doc for this. IMHO, it does not make sense to repeat Stream there as it is stated at the beginning and just below. `Stream implements mino.RPC. It ...` is enough to get all the information in one place.

Reply:
Then we should change the phrasing, because `Stream implements mino.RPC.Stream` describes precisely what it does, while saying `Stream implements mino.RPC` is not exactly right.
To be right it should be something like `Stream implements one of the function described by mino.RPC.Stream`.

Reply:
That's exactly what `Stream implements mino.RPC.` means. You implement an interface by implementing the functions of the interface. Yes it's one of the functions, but you know which one.

Reply:
> You implement an interface by implementing the functions of the interface

True for a struct, not a function. A struct can say that it implements an interface, but a function not, this is why I am not comfortable saying `Stream implements mino.RPC.`. (and this is why in the first place I was using "()", to indicate that the abuse of saying "blabla implements interface.blabla()" actually means this is *contributing to* implementing  the interface, but does not actually *implements* it, since it's a function.)

Anyway, we can agree to disagree on this one and keep going with your proposed phrasing.